package.json

```diff
@@ -33,7 +33,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "4.12.0",
     "@typescript-eslint/parser": "4.12.0",
-    "eslint": "7.17.0",
+    "eslint": "7.18.0",
```

Comment:
> __[Goodcheck]__ Don't forget updating `peerDependencies` when updating `devDependencies`. ([view](https://sider.review/gh/repos/72121459/pulls/859#issue-5307165))
>
> <details>
>
> | Rule |
> | ---- |
> | `update-peer-dependencies` |
>
> You can [close this issue](https://sider.review/gh/repos/72121459/pulls/859#issue-5307165) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue, because:
- When not updating `peerDependencies`.